website/docs/automation/gitops/argocd/cluster_bootstrap.md

```diff
@@ -45,19 +43,18 @@ $ git -C ~/environment/argocd push --set-upstream origin main
 Create an Argo CD secret to give access to the Git repository from Argo CD:
 
 ```bash
-$ argocd repo add $(cat ~/environment/argocd_repo_url) --ssh-private-key-path ${HOME}/.ssh/gitops_ssh.pem --insecure-ignore-host-key --upsert --name git-repo
+$ argocd repo add $GITOPS_REPO_URL_ARGOCD --ssh-private-key-path ${HOME}/.ssh/gitops_ssh.pem --insecure-ignore-host-key --upsert --name git-repo
 Repository 'ssh://APKA232DOTYWDIYNGJC6@git-codecommit.eu-central-1.amazonaws.com/v1/repos/eks-workshop-argocd' added
 ```
 
 Argo CD application is a CRD Kubernetes resource object representing a deployed application instance in an environment. It defines key information about the application, such as the application name, the Git repository, and the path to the Kubernetes manifests. The application resource also defines the desired state of the application, such as the target revision, the sync policy, and the health check policy.
 
-As the next step let's create an Argo CD Application which automatically `Sync` application with desired state in the Git repository:
+As the next step let's create an Argo CD Application which `Sync` with desired state in the Git repository:
 
 ```bash
-$ argocd app create apps --repo $(cat ~/environment/argocd_repo_url) \
-  --path apps --dest-server https://kubernetes.default.svc \
-  --sync-policy automated --self-heal --auto-prune
- application 'apps' created
+$ argocd app create apps --repo $GITOPS_REPO_URL_ARGOCD \
```

Comment:
I think we need to add the `--auto-prune` to be able to clean up the app during `reset-environment` when it deletes the app.

Reply:
Added --sync-policy automated --self-heal --auto-prune and AutoSync for the App of Apps